app/emoji/client/emojiParser.js

```diff
@@ -18,7 +18,7 @@ Tracker.autorun(() => {
 	callbacks.add('renderMessage', (message) => {
 		if (s.trim(message.html)) {
 			// &#39; to apostrophe (') for emojis such as :')
-			message.html = message.html.replace(/&#39;/g, '\'');
+			message.html = s.trim(message.html.replace(/&#39;/g, '\''));
```

Comment:
please change to either save the `trim` value within this function or add a `trim` on `renderMessage` call.